packages/atd-windows.1.2.0/opam

```diff
@@ -0,0 +1,13 @@
+opam-version: "1.2"
+maintainer: "pierre.boutillier@laposte.net"
+authors: "Martin Jambon"
+homepage: "https://github.com/mjambon/atd"
+bug-reports: "https://github.com/mjambon/atd/issues"
+dev-repo: "https://github.com/mjambon/atd.git"
+license: "BSD3"
+patches: ["patches/ocamlc_config_through_ocamlfind.patch"]
+build: ["env" "OCAMLFIND_TOOLCHAIN=windows" make]
+install:
+["env" "OCAMLFIND_TOOLCHAIN=windows" make "install" "BINDIR=%{bin}%"]
```

Comment:
this *definitely* shouldn't be `%{bin}%`, rather it should be within the ocaml-windows root